app/tests/unit/forms/form/routes.spec.js

```diff
@@ -0,0 +1,120 @@
+const request = require('supertest');
+const { v4: uuidv4 } = require('uuid');
+
+const { expressHelper } = require('../../../common/helper');
+
+const apiAccess = require('../../../../src/forms/auth/middleware/apiAccess');
+const userAccess = require('../../../../src/forms/auth/middleware/userAccess');
+const rateLimiter = require('../../../../src/forms/common/middleware/rateLimiter');
+const validateParameter = require('../../../../src/forms/common/middleware/validateParameter');
+const controller = require('../../../../src/forms/form/controller');
+
+//
+// Mock out all the middleware - we're testing that the routes are set up
+// correctly, not the functionality of the middleware.
+//
```

Comment:
The previous "route" tests were actually testing the controllers. These tests will check that middleware is being called, which mostly just confirms that routes are configured properly (the middleware is tested in its own unit tests).